apis/apis.json

```diff
@@ -5071,4 +5073,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
I suspect it's worth removing this trailing newline otherwise it'll show up again when it's regenerated for some reason.